frontend/src/app/commonComponents/RadioGroup.tsx

```diff
@@ -5,7 +5,12 @@ import Required from "./Required";
 import Optional from "./Optional";
 
 type OptionsKeys = { [label: string]: string };
-type OptionsArray = { label: string; value: string; disabled?: boolean }[];
+type OptionsArray = {
+  label: string;
```

Comment:
Could make this `label: React.ReactNode`; see below.

Reply:
Done in https://github.com/CDCgov/prime-simplereport/pull/869/commits/25989e3ec6fcbe049a449f4c4ab6f44fd7d2e5d8